pkgs/tools/text/rosie/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, lib
+, fetchgit
+, libbsd
+, readline
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rosie";
+  version = "unstable-2020-01-11";
+  src = fetchgit {
```

Comment:
Perhaps replace `fetchgit` with `fetchFromGitLab`, as that's generally faster than a `git clone` as `fetchgit` does.

Reply:
Can't, need to fetch submodules, fetchFromGitLab doesn't do that

Reply:
I did have an alternative, which used fetchFromGitLab/fetchFromGitHub with the submodules, perhaps I should resurrect it: https://github.com/KoviRobi/nixpkgs/commit/0dcbc21569fa58911ff29858875e4de6b7d3fe16